### PR TITLE
Add freezable filter

### PR DESCRIPTION
**Pull request by TheoBrigitte**
This PR adds the capability to check if owner can freeze tokens.

I added this check within the actual renounced filter in order to avoid to duplicate `getAccountInfo` RPC call.

### CODE REVIEW

**Comment by fdundjer** [comment]
@TheoBrigitte Can you update readme?

**Comment by TheoBrigitte** [comment]
> @TheoBrigitte Can you update readme?

done

File: filters/renounced.filter.ts [hunk 1]
```diff
@@ -4,23 +4,29 @@ import { Connection } from '@solana/web3.js';
...
+export class RenouncedFreezeFilter implements Filter {
+  constructor(private readonly connection: Connection, private readonly checkRenounced: boolean, private readonly checkFreezable: boolean) {}
 
   async execute(poolKeys: LiquidityPoolKeysV4): Promise<FilterResult> {
+    const errorMessage = [ this.checkRenounced ? 'mint' : undefined, this.checkFreezable ? 'freeze' : undefined ].filter((e) => e !== undefined);
     try {
       const accountInfo = await this.connection.getAccountInfo(poolKeys.baseMint, this.connection.commitment);
       if (!accountInfo?.data) {
-        return { ok: false, message: 'Renounced -> Failed to fetch account data' };
+        return { ok: false, message: 'Failed to fetch account data' };
```
Comment (fdundjer) [inline filters/renounced.filter.ts:15]: Can we keep filter prefix here?

Comment (TheoBrigitte) [reply filters/renounced.filter.ts]: yes sure

**Pull request merged**